tools/cli/beeAI.py

```diff
@@ -0,0 +1,43 @@
+#!/usr/bin/env python3
```

Comment:
What package is this going to go into?

We currently have a single package, 'bee_hive'. We could add the cli into this, or add a new package.
In both cases the files should sit under the poetry definitions

(need to look at best practices for structuring a cli)

Could be a part 2, but I do think ensuring we package will make it easier to then depend/consume from other places, including demos , as well as get consistent behaviour and add testing within a framework. (these tests can also be managed by pytest within the module structure)